connaisseur/validators/notaryv1/notary.py

```diff
@@ -89,49 +89,58 @@ def healthy(self):
+        image: Image,
+        role: TUFRole,
+        token: str = None,
+    ):
         im_repo = f"{image.repository}/" if image.repository else ""
         url = (
             f"https://{self.host}/v2/{image.registry}/{im_repo}"
             f"{image.name}/_trust/tuf/{str(role)}.json"
         )
 
-        async with aiohttp.ClientSession() as session:
-            request_kwargs = {
-                "url": url,
-                "ssl": self.cert,
-                "headers": ({"Authorization": f"Bearer {token}"} if token else None),
-            }
-            async with session.get(**request_kwargs) as response:
-                status = response.status
-                if (
-                    status == 401
-                    and not token
-                    and ("www-authenticate" in [k.lower() for k in response.headers])
-                ):
-                    auth_url = self.__parse_auth(
-                        {k.lower(): v for k, v in response.headers.items()}[
-                            "www-authenticate"
-                        ]
-                    )
-                    token = await self.__get_auth_token(auth_url)
-                    return await self.get_trust_data(image, role, token)
-
-                if status == 404:
-                    msg = "Unable to get {tuf_role} trust data from {notary_name}."
-                    raise NotFoundException(
-                        message=msg, notary_name=self.name, tuf_role=str(role)
-                    )
-
-                response.raise_for_status()
-                data = await response.text()
-                return TrustData(json.loads(data), str(role))
+        request_kwargs = {
+            "url": url,
+            "ssl": self.cert,
+            "headers": ({"Authorization": f"Bearer {token}"} if token else None),
+        }
+        async with session.get(**request_kwargs) as response:
+            status = response.status
+            if (
+                status == 401
+                and not token
+                and ("www-authenticate" in [k.lower() for k in response.headers])
+            ):
+                auth_url = self.__parse_auth(
+                    {k.lower(): v for k, v in response.headers.items()}[
+                        "www-authenticate"
+                    ]
+                )
+                token = await self.__get_auth_token(session, auth_url)
+                return await self.get_trust_data(session, image, role, token)
+
+            if status == 404:
+                msg = "Unable to get {tuf_role} trust data from {notary_name}."
+                raise NotFoundException(
+                    message=msg, notary_name=self.name, tuf_role=str(role)
+                )
+
+            response.raise_for_status()
+            data = await response.text()
+            return TrustData(json.loads(data), str(role))
 
     async def get_delegation_trust_data(
-        self, image: Image, role: TUFRole, token: str = None
+        self,
```

Comment:
this might be an old version of black

Reply:
It doesn't fail black validation, so I don't think so. Also it got bigger with another argument, so it makes sense it's no longer a single line

Reply:
ok, then let's keep it